aea/configurations/base.py

```diff
@@ -1783,7 +1783,7 @@ def _get_contract_interfaces(self) -> Dict[str, str]:
                 with open(full_path, "r") as interface_file_ethereum:
                     contract_interface = json.load(interface_file_ethereum)
                     contract_interfaces[identifier] = contract_interface
-            elif identifier == "cosmos":
+            elif identifier in ["cosmos", "fetchai"]:
```

Comment:
(not related to this PR): Eventually, also these magic values should be replaced.

Reply:
yes..there is a circular import here if I replace now..we need to clean it